### ISSUE TITLE
ARM support

### ISSUE THREAD

**Issue by morganchristiansson** [kedacore/http-add-on#364]
Same as kedacore/keda#779

Release multi-arch docker images with arm64 support.

Should finish kedacore/keda#779 first, creating this as placeholder as there's no existing issue.

**Comment by arschles** [comment]
Thanks @morganchristiansson.

cc/ @tomkerkhove

**Comment by JorTurFer** [comment]
Nowadays, KEDA supports ARM64 arch, so I can tackle this issue and do the same as I did there.
Do you think it's useful @arschles ?

**Comment by arschles** [comment]
@morganchristiansson do you still have a use for this? If so, then @JorTurFer I do think it's useful.

**Comment by morganchristiansson** [comment]
Yeah my home cluster is multi-architecture with Raspberry Pis and 1x fileserver Intel node. But I prefer to run everything on Pis/arm64.

Also I think arm64 is useful for all Raspberry Pi clusters, EKS with AWS Graviton nodes, etc out there

**Comment by chairwa** [comment]
kedacore/keda-add-ons-http not support arm64 architecture yet ?

**Comment by JorTurFer** [comment]
No it doesn't. The project has been a bit frozen last months:  https://github.com/kedacore/http-add-on/discussions/523
Once it starts again, I added e2e tests to ensure the behavior, but I didn't add ARM64 support yet

**Comment by chairwa** [comment]
I'm new to keda now.
